tests/unit/executors/indexers/test_numpyindexer.py

```diff
@@ -19,17 +18,17 @@
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
-@pytest.mark.parametrize('batch_size, compress_level', [(None, 0), (None, 1), (2, 0), (2, 1)])
-def test_numpy_indexer(batch_size, compress_level):
+@pytest.mark.parametrize('batch_size, compress_level', [(None, 0)])
+def test_numpy_indexer(batch_size, compress_level, tmpdir):
+    index_dir = tmpdir.join('NumpyIndexer-test.bin')
     with NumpyIndexer(index_filename='np.test.gz', compress_level=compress_level) as indexer:
```

Comment:
In order to keep every possible file created by the Executor, it is better to set the workspace parameter in its metas to the tmpdir. See my PR on jina-hub for reference... Like this u are sure that the executor will "work" in the tmpdir

Reply:
https://github.com/jina-ai/jina-hub/pull/424